src/app/App.js

```diff
@@ -118,6 +107,7 @@ function renderNavMenu(parent, activeItemNr = 0, previousState = undefined) {
 }
 
 const startGame = () => console.log('witaj w grze');
+const chosenAnswer = (...params) => console.log(params);
```

Comment:
Nie wiem czemu, ale to mi wyglada na `chosenAnswer` wybrana odpowiedź, a nie wybrano odpowiedź. W sumie startGame podobnie. Może zmienić to na `answerWasChosen` :) ? Albo chociaż `asnwerChosen` albo `onAnswerChosen` / `onChooseAnswer`?. Na górze analogicznie np. `onGameStarted`

Reply:
Mogę się wzorować na klasie Player - onQuestionAnswered będzie ok? Czy lepiej nie stosować tej samej nazwy, jak nie jestem pewna czy faktycznie dotyczy tego samego, żeby nie wprowadzać nikogo w błąd?

Reply:
Ja w sumie zastosowałem kiedyś to robiąc i mialem problem. Moze walsnie warto powiedziec, ze tutaj to dopiero wybor odpowiedzi :)

Reply:
Done